shared/actions/platform-specific/index.desktop.tsx

```diff
@@ -230,7 +232,7 @@ const sendKBServiceCheck = (state: Container.TypedState, action: ConfigGen.Daemo
     state.config.daemonHandshakeWaiters.size === 0 &&
     state.config.daemonHandshakeFailedReason === ConfigConstants.noKBFSFailReason
   ) {
-    SafeElectron.getApp().emit('keybase' as any, {type: 'requestStartService'})
```

Comment:
likely means windows wasn't starting the service

Reply:
cc: @xgess

Reply:
it was. on my windows machine, the gui is definitely starting the service (i think via this mechanism). and actually, i have a PR up to give that a little tweak: https://github.com/keybase/client/pull/22023/files#diff-27d3a4d2bf9aa2500837b33a82f688a0R83